services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -1963,7 +1962,7 @@ export class WorkflowService {
           endUserService: this.endUserService,
           projectId: currentProjectId,
           businessId: workflowRuntimeData.businessId,
-          sendEvent: service.sendEvent,
+          sendEvent: e => service.sendEvent(e),
```

Comment:
```suggestion
          sendEvent: (event) => service.sendEvent(event),
```

Reply:
There is already an `event` variable in the scope, that's why I picked `e`, as we just pass it along and it's common practice.